app/src/main/java/net/programmierecke/radiodroid2/StationSaveManager.java

```diff
@@ -304,7 +304,7 @@ public static String getSaveDir() {
     }
 
     public void SaveM3U(final String filePath, final String fileName) {
-        Toast toast = Toast.makeText(context, "Writing to " + filePath + "/" + fileName + ".. Please wait..", Toast.LENGTH_SHORT);
+        Toast toast = Toast.makeText(context, context.getResources().getText(R.string.notify_save_playlist_now, filePath, fileName), Toast.LENGTH_LONG);
```

Comment:
`getString` instead of `getText` in all cases. The text is not styled. and `getText` doesn't support format arguments. With this change it works with both languages.